tests/smoke-tests/test-balances-consistency.ts

```diff
@@ -25,7 +25,7 @@ describeSmokeSuite(`Verifying balances consistency...`, (context) => {
   let specVersion: number = 0;
   let runtimeName: string;
 
-  before("Retrieve all balances", async function () {
+  before(`Retrieve all balances`, async function () {
```

Comment:
Any reason to change the double quotes to backquotes ?
If this is something we want to implement, we should make sure it is in the prettier configuration

Reply:
hang over from previous iteration when it used string templates for suite ID inserted in